daml-triggers/daml/Trigger/AutoApprove.daml

```diff
@@ -36,8 +36,7 @@ autoApprove : Party -> TriggerA () ()
 autoApprove sender = do
   blackLists <- query @Blacklist
 
-  when (1 /= length blackLists) $ error "There should be only one Blacklist"
-  let (_, blacklist) = head blackLists
+  let blacklist = snd <$> head blackLists
```

Comment:
@tamaskalcza-da 
I would actually argue that one should either take all blacklists into account by merging them or allow just one (or 0).

Like this I feel like it's too easy to not realize that there already exists a blacklist, you create a new one, and then more or less randomly one of the two is picked.

Reply:
When I checked the models it seemed to me that it is not possible to have more than one `Blacklist` contracts. Am I wrong? 🤔

Reply:
My understanding was that we should not really on keys to ensure uniqueness

Reply:
Isn't this the case with Canton in multi-domain setup? Even so, I get your point and will make the change.